azure-pipelines/cd.yml

```diff
@@ -1,64 +1,174 @@
-trigger:
-  tags:
-    include:
-    - v*
-
+# Manual trigger only
+trigger: none
 pr: none
 
-strategy:
-  matrix:
-    linux:
-      imageName: 'ubuntu-16.04'
-    mac:
-      imageName: 'macos-10.14'
-    windows:
-      imageName: 'vs2017-win2016'
-
-pool:
-  vmImage: $(imageName)
+parameters:
+- name: buildAgentPoolVar
+  displayName: 'Build agent pool'
+  type: string
+  default: 'BuildAgentPool'
+- name: buildAgentVmImageVar
+  displayName: 'Build agent image'
+  type: string
+  default: 'BuildAgentImage'
+- name: ghRelease
+  displayName: 'Draft github release'
+  type: boolean
+  default: false
+- name: isPreRelease
```

Comment:
Placeholder comment capturing discussion to have explicit to prod parameter/flag where when not set (default) publish to marketplace is pre-release.